starters/basic-starter/components/meta.tsx

```diff
@@ -0,0 +1,95 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { DrupalMetatag } from 'types/drupal';
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  tags?: DrupalMetatag[];
+}
+
+export function Meta({ tags }: MetaProps) {
+  const router = useRouter();
+  const origin =
+    typeof window !== 'undefined' && window.location.origin
+      ? window.location.origin
+      : '';
+  const currentUrl = `${origin}${router.asPath !== '/' ? router.asPath : ''}`;
+
+  // Initialize imageUrl variable with default value.
+  let imageUrl = `${origin}/_next/image?url=`;
+  // Check whether the tags is an array list.
+  if (Array.isArray(tags)) {
+    // Filter out image property from og:image
+    const filterImage = tags.filter((obj) => {
```

Comment:
@rajeshreeputra Why are we looping here ? Can we directly add image url before [here](https://github.com/acquia/next-acms/pull/273/files#diff-7c59aaf9bf3aa0972d9706953527a211889939dbc700d60d4eec3659a0a5d3cdR50) ?

Reply:
```suggestion
    tags = tags.filter((obj) => {
        if ( 'og:image' === obj.attributes.property) {
          obj.attributes.content = `obj.attributes.content${encodeURIComponent(imageSrc)}&w=1200&q=75`
        }
      return obj;
    });
```